File: java/src/main/java/com/touzhijia/platform/entity/ServiceData.java
package com.touzhijia.platform.entity;

import com.google.gson.JsonElement;

public class ServiceData {
	// 业务标识
	private String service;
	// 业务请求或响应
	private JsonElement body;
	
	public ServiceData(String service, JsonElement body) {
		this.service = service;
		this.body = (JsonElement) body;
	}
	
	public String getService() {
		return service;
	}
	
	public JsonElement getBody() {
		return body;
	}
}

File: c#/touzhijia/entity/ServiceData.cs
using Newtonsoft.Json.Linq;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace touzhijia.entity
{
    [Serializable]
    public class ServiceData
    {
        public ServiceData()
        {
        }
        public ServiceData(string service, Object body)
        {
            this.service = service;
            this.body = body;
        }

        public string service { get; set; }
        public Object body { get; set; }
    }
}
